### ISSUE TITLE
Can't eval defprotocol in cljs source file

### ISSUE THREAD

**Issue by mobbjelly** [LightTable/LightTable#2282]
**Reproduce**

1.  Create project using `Lein new mies`
2.  Add some code in core.cljs
3.  Eval code by hitting ctrl + enter
4.  Error occurs in defprotocol

![error](https://cloud.githubusercontent.com/assets/8497784/19139199/0961a6a2-8bb5-11e6-943e-1a8bca014e8d.png)

**Environment**

-  Windows 10 64bit
-  LT 0.8.1
-  Default user.keymap and user.behaviors
-  Default plugins (Didn't download any external plugins)

**Comment by sbauer322** [comment]
To clarify, with `Lein new mies` do you mean to use the mies template and not a new default project named mies?

**Comment by mobbjelly** [comment]
Yeah, I use the [mies template](https://github.com/swannodette/mies)

**Comment by mobbjelly** [comment]
Relevant info for refer: [defprotocol fails oddly in repl *only*](https://groups.google.com/forum/#!topic/clojurescript/__Qf6L40cwU)

**Comment by sbauer322** [comment]
I am able to reproduce this, so at least there is that. 

Curiously, if you adjust the project to be Clojure rather than ClojureScript it evals correctly.

Unfortunately, my guess as to the actual problem would be the internal version of ClojureScript that Light Table uses for a repl does not have the fix reported in your link... or otherwise some other incompatibility. We have issue #1973 to address the ClojureScript update and intend to start the update this month as it has become too significant an issue to ignore. 

I could very well be incorrect, but I can't think of another reason for the problem you are experiencing... though perhaps someone else will.
